doc/releases.md

```diff
@@ -0,0 +1,51 @@
+# Releases
+
+https://github.com/badges/shields is a monorepo and hosts the Shields frontend and server code as well as the [badge-maker][npm package] NPM library (and the [badge design specification](https://github.com/badges/shields/tree/master/spec)). The packaging and release processes for these respective items is described below.
+
+## badge-maker package
+
+We follow [Semantic Versioning](https://semver.org/) for the badge-maker package, and publish git Tags using the form `X.Y.Z`, and all such tags of this form are badge-maker releases (e.g. [3.3.0](https://github.com/badges/shields/releases/tag/3.3.0)
+
+The [badge-maker][npm package] is also published to the npm.js registry.
+
+Releases of badge-maker are done as and when needed, and not on any predetermined interval.
+
+## Shields.io service
+
+The [Shields.io Service][shields.io] consists of the frontend [https://shields.io][shields.io] website which allows users to browse and discover available badges, as well as the badge server backend that serves up requested badges (e.g. https://img.shields.io/badge/badges-rock-blue).
+
+This is the core, free, anonymous service available for anyone to use and which serves up more than 500 million badges every month!
+
+We do not have a fixed schedule for deploying updates to the Shields.io production environment, but we typically deploy the latest version at least once per week.
+
+More information about the production environment can be found [here][production hosting]
+
+## Shields server
+
+Some users may wish to host their own instance of Shields so we also make it possible for users to do so. This is particularly useful if you want to serve badges for resources that require authentication or are not exposed to the internet (e.g: inside a corporate network). A variety of options are available either by installing from source or using a docker image.
+
+### Important information
+
+This Shields server is the exact same codebase that powers the main Shields.io service; we do not have a separate self-hosted version of Shields. This means that the server codebase is geared towards the development, maintenance, and operation of the Shields.io service so there are a few things to note:
+
+- We often have to reject or de-prioritize feature requests that are only applicable for self-hosting and/or which may be problematic for the core Shields.io offering (e.g. requiring auth on the badge server)
+- We do not accept new badges that cannot be utilized with Shields.io (e.g. those that would always require user-specific authorization)
+- We do not do any additional testing nor validation of self-hosted instances (e.g. we test Shields.io with Jira Cloud badges, but we don't test a self-hosted Shields server against a self-hosted Jira server instance with auth)
+- We're happy to try to offer some tips and guidance to self-hosters when and where we can, but we don't have the ability to troubleshoot/debug/support/etc. self-hosted Shields servers
```

Comment:
Some of the wording I came up with yesterday, maybe to incorporate somewhere around here:

> Spotted something wrong? Why not step up and help us fix it by submitting a pull request? All community contributions, even documentation improvements, are welcome!

Reply:
I really like the theme of this as we discussed in the previous Ops meeting, so I elected to pull this up to the very top of the doc since it's really applicable to the entirety of the Shields project (not just self-hosting). In doing so I did tweak the wording _slightly_ so please let me know what you think!